cryosmsSup/cryosms.proto

```diff
@@ -52,11 +52,13 @@ getRampTarget {
     wait 100;
 }
 
+getHoldTarget {
+    in "%*8c RAMP STATUS: HOLDING ON TARGET AT %f %(\$1){AMPS|TESLA}";
+    wait 100;
+}
+
```

Comment:
This has no `out` so i was curious what was actioning the reply it reads. It seems to come from a Db triggered from c++, so i guess that must action something to cause the output it reads?

Reply:
It's an asynchronous message that gets caught by an io/intr record, and then that reading gets used from the C++ side

Reply:
They get sent in response to something initiated from c++ side e.g. starting a ramp, or they just sent periodically anyway by the hardware?

Reply:
My understanding is that it gets sent asynchronously after a ramp completes and the PSU considers that it's "got there". So in a sense yes it is "triggered" from the C++ side, by starting a ramp, For ZF mode ramps will be relatively quick as it's making small changes. For a full-field ramp it could take hours, depending on ramp rates.

Reply:
close, this is the reply to "RAMP STATUS" if it's holding on target. Just using it here to get the RBV for what the device thinks the target of the last ramp it did was. Needed for detecting small, quick ramps.